index.bs

```diff
@@ -9034,7 +9034,7 @@ Attributes</h4>
 The {{StereoPannerNode}} Interface</h3>
 
 This interface represents a processing node which positions an
-incoming audio stream in a stereo image using a low-cost <a href="#Spatialization-equal-power-panning">equal-power panning
+incoming audio stream in a stereo image using <a href="#stereopanner-algorithm">a low-cost equal-power panning
```

Comment:
Maybe we should just leave out "equal-power"?  If so, we should also remove "equai-power" from the API Overview for the stereo panner.